src/main/java/frc/robot/commands/PrepCommand.java

```diff
@@ -7,24 +7,35 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.Shooter;
 
-public class AmpCommand extends Command {
-  
+public class PrepCommand extends Command {
+ 
   private final Shooter shooter;
-  private double speed = 1; // TODO: Change to amp motor speed (in rotations per second)
-  private final double ampAlignAngle = 36; // TODO: tune relative to shooting from right in front of amp
+  private double angle = 0;
+  private double speed = 0;
 
-  public AmpCommand(Shooter shooter) {
+  /** Creates a new PrepCommand. */
+  public PrepCommand(Shooter shooter, double angle, double speed) {
+    // Use addRequirements() here to declare subsystem dependencies.
     this.shooter = shooter;
-    addRequirements(shooter);
```

Comment:
Noticed this was removed. You still want to do addRequirements in a command passing in a subsystem. What this ensures is that you cannot have more than one command controlling a subsystem at any given time.

Reply:
My new pr should fix this, I think it will work when we just merge that one in